src/lib/core/fiber.h

```diff
@@ -584,8 +574,20 @@ struct fiber {
 	struct fiber_slice max_slice;
 	/** Valgrind stack id. */
 	unsigned int stack_id;
-	/* A garbage-collected memory pool. */
+	/** A garbage-collected memory pool. */
 	struct region gc;
+#ifdef ENABLE_BACKTRACE
+	/**
+	 * Backtrace of the first fiber gc allocation that found
+	 * leaked on inspection.
+	 */
+	struct backtrace *leak_bt;
+#endif
+	/**
+	 * This much size of fiber gc at the beginning is used for
+	 * fiber internal purpuses.
+	 */
+	size_t gc_internal;
```

Comment:
Better name it `gc_initial_size` or something like this then.

Reply:
Done.